modules/arm_plugin/src/transformations/store_result_name.cpp

```diff
@@ -6,7 +6,7 @@
 #include "store_result_name.hpp"
 #include "transformations/utils/utils.hpp"
 
-NGRAPH_RTTI_DEFINITION(ArmPlugin::pass::StoreResultName, "StoreResultName", 0);
+NGRAPH_RTTI_DEFINITION(ArmPlugin::pass::StoreResultName, "StoreResultName");
```

Comment:
@ilyachur we remove the version here, but the macro `NGRAPH_RTTI_DEFINITION` define in OpenVINO doesn't provide the initialization for two in put parameter.

Reply:
You shouldn't use `NGRAPH_RTTI_DEFINITION` please use `OPENVINO_OP`